### ISSUE TITLE
request for supporting ReST filter

### ISSUE THREAD

**Issue by arcturusannamalai** [GNUAspell/aspell#641]
ReST mode is useful to write documentation for Python code as well as books for technical subjects in their own regard; see: https://docutils.sourceforge.io/docs/user/rst/quickstart.html

Specifically, the spell mode for ReST could skip over,
- code sections
- standard meta information / keywords in ReST
- etc.

**Comment by kevina** [comment]
I agree that a ReST filter will be useful.  The existing markdown filter would be a good place to start.
